platforms/documentation/docs/src/docs/userguide/core-plugins/checkstyle_plugin.adoc

```diff
@@ -92,13 +89,48 @@ The Checkstyle plugin adds the following dependency configurations:
 | The Checkstyle libraries to use
 |===
 
+By default, the checkstyle plugin contains a dependency on `com.puppycrawl.tools:checkstyle`.
```

Comment:
I think this phrasing may be confusing, as the plugin doesn't exactly contain a dependency. It would be clearer to say the `checkstyle` configuration contains it.